### ISSUE TITLE
Repetitive vault access confirmation popup on Mac OS Sonoma v.14.13 for Desktop v.2024.2.0

### ISSUE THREAD

**Issue by myvocabu** [bitwarden/clients#8038]
### Steps To Reproduce

Popup (![Bitwarden](https://github.com/bitwarden/clients/assets/96499525/6e8ca20e-a837-48cc-a8f4-2a47843b3a20)) with confirmation access to MacOS vault shows every 10-15 min on the Desktop after upgrade to fresh version Version 2024.2.0

### Expected Result

Popup apiaries only once or at least by configured time out 

### Actual Result

Popup randomly shows from the backgroud regardless if app is used

### Screenshots or Videos

![Bitwarden](https://github.com/bitwarden/clients/assets/96499525/fe4ab995-9a57-4cd0-9770-eeaf5b056e0f)
![Bitwarden](https://github.com/bitwarden/clients/assets/96499525/fb4e7de8-1acd-4a8a-a597-61ac4da1c458)
![MacBook Pro](https://github.com/bitwarden/clients/assets/96499525/b05bf6cb-e92e-427d-ac16-cec065121149)

### Additional Context

Version 2024.2.0
Shell 28.1.3
Renderer 120.0.6099.199
Node 18.18.2
Architecture arm64

### Operating System

macOS

### Operating System Version

14.13

### Installation method

Mac App Store

### Build Version

Version 2024.2.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by NovaSilentium** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
